tests/utils/test_env_checker.py

```diff
@@ -115,95 +116,120 @@ def test_check_reset_seed(test, func: callable, message: str):
             check_reset_seed(GenericTestEnv(reset_fn=func))
 
 
-def _reset_var_keyword_kwargs(self, kwargs):
-    return self.observation_space.sample()
-
-
-def _reset_return_info_type(self, seed=None, return_info=False, options=None):
+def _deprecated_return_info(self, return_info=False):
```

Comment:
What exactly is the purpose of this?

Reply:
(some comments/description, and type hints)

Reply:
The idea is we want to throw a warning if `return_info` is in the signature of an environment's reset function

Reply:
Yea, but these specific functions are extremely odd. Please at least annotate these tests so that it's clear to us 6 months from now what these functions are meant to do exactly, or perhaps restructure the tests so that it's more obvious.

Reply:
ah, I missed your second comment, yeah I can add some more explanation of the specific reason for the deprecation test, as well as type hints.